.github/workflows/doc_links_checker.yml

```diff
@@ -0,0 +1,28 @@
+name: Doc Link Checker
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  linkchecker:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Load Excludes
+      run: |
+        LYCHEE_EXCLUDE=$(sed -e :a  -e 'N;s/\n/ --exclude /;ta' .lycheeexclude)
+        echo "LYCHEE_EXCLUDE=$LYCHEE_EXCLUDE" >> $GITHUB_ENV
+    - name: Lychee Link Checker
+      id: lychee
+      uses: lycheeverse/lychee-action@master
+      with:
+        args: --accept=200,403,429 --exclude ${{ env.LYCHEE_EXCLUDE }} --exclude-mail "**/*.html" "**/*.md" "**/*.txt" "**/*.json" "**/*.js" "**/*.ts" "**/*.tsx"
+      env:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    - name: Fail if there were link errors
+      run: exit ${{ steps.lychee.outputs.exit_code }}
```

Comment:
are we confident enough that we want it to fail if this link checker fails. we could always just output an error message and succeed. but if we are confident after the excludes list then good for us!

Reply:
I personally prefer to fail the checker for the similar reason with lint checker. It maybe not a serious bug sometimes but usually is a defect. People are born to ignore warnings 🤣